tests/test_transforms.py

```diff
@@ -1,94 +1,113 @@
 """Tests for transforms.py module."""
+
 import numpy as np
 import pytest
 
-from distrx.transforms import transform_data, delta_method
-
+from distrx.transforms import (
+    delta_method,
+    transform_data,
+    transform_percentage_change,
+    transform_percentage_change_counts,
+)
 
 TRANSFORM_DICT = {
-    'log': [
-        np.log,
-        lambda x: 1.0/x
-    ], 'logit': [
-        lambda x: np.log(x/(1.0 - x)),
-        lambda x: 1.0/(x*(1.0 - x))
-    ], 'exp': [
-        np.exp,
-        np.exp
-    ], 'expit': [
-        lambda x: 1.0/(1.0 + np.exp(-x)),
-        lambda x: np.exp(-x)/(1.0 + np.exp(-x))**2
-    ]
+    "log": [np.log, lambda x: 1.0 / x],
+    "logit": [lambda x: np.log(x / (1.0 - x)), lambda x: 1.0 / (x * (1.0 - x))],
+    "exp": [np.exp, np.exp],
+    "expit": [
+        lambda x: 1.0 / (1.0 + np.exp(-x)),
+        lambda x: np.exp(-x) / (1.0 + np.exp(-x)) ** 2,
+    ],
 }
 TRANSFORM_LIST = list(TRANSFORM_DICT.keys())
 FUNCTION_LIST = [transform_data, delta_method]
-VALS = [0.1]*2
+VALS = [0.1] * 2
 
 
-@pytest.mark.parametrize('transform', TRANSFORM_LIST)
+@pytest.mark.parametrize("transform", TRANSFORM_LIST)
 def test_method_name_valid(transform):
     """Raise ValueError for invalue `method`."""
     with pytest.raises(ValueError):
-        transform_data(VALS, VALS, transform, method='dummy')
+        transform_data(VALS, VALS, transform, method="dummy")
```

Comment:
We recently started using pydantic for checking input. Not top priority, but if we used pydantic we could get rid of these sorts of tests!